src/utils.ts

```diff
@@ -0,0 +1,28 @@
+import * as execa from "execa";
+
+export const createExecaCommand =
+  (command: string, config: {}) => async (params: string[]) =>
+    execa(command, params, config);
+
+export const checkDumpName = (name: string) => {
```

Comment:
I think we could give this a more descriptive name, we used it in the delete and unpublish work but had check what it was doing before figuring out which code we wanted to execute

What do you think of something like `checkDumpNameForTag`? That way it's easier to expect what a `true` result is

Reply:
I'm cool with the suggested name: changed!